src/OMSimulatorLib/SystemWC.h

```diff
@@ -71,7 +71,7 @@ namespace oms3
 
   private:
     std::string solverName = "oms-ma";
-    double time;
+    double time=0;
```

Comment:
This doesn't make too much sense ...

Reply:
What I mean is, it should be initialized with `startTime`, and therefore it can only be initialized inside the initialize method.

Reply:
I agree. But Valgrind for some reason gives a "Conditional jump or move depends on uninitialised value(s)" error without the default value, even when the variable has in fact been initialized.